README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Dependencies
 
-* [Isabelle2023](https://isabelle.in.tum.de/website-Isabelle2023/dist/Isabelle2023_linux.tar.gz)
+* [Isabelle2024](https://isabelle.in.tum.de)
```

Comment:
Please use https://isabelle.in.tum.de/website-Isabelle2024/dist/Isabelle2024_linux.tar.gz